docs/current_docs/api/services.mdx

```diff
@@ -296,6 +296,64 @@ dagger call services up --ports 8080:80
 </TabItem>
 </Tabs>
 
+## Use service endpoints
+
+Every service has an endpoint, and this endpoint can be obtained via the Dagger API. The following example demonstrates starting an HTTP service manually, retrieving its endpoint, and then querying that service using a standard HTTP GET request:
```

Comment:
@vikram-dagger would it be helpful to also include a specific use case of how/why you would use this?

Reply:
@marcosnils can you comment on this?